auth_proxy/models/oauth.py

```diff
@@ -68,7 +70,9 @@ class Grant(Base):
     _scopes = Column('scopes', Text)
 
     def delete(self):
-        self.expires = 0
+        self.expires = datetime.now()
```

Comment:
The following isn't a suggestion, but rather a "do I understand this" question: who calls this `delete`, and can *they* pass in the dependency or call save on the session?